contract/Cargo.toml

```diff
@@ -16,9 +16,11 @@ near-sdk = { workspace = true }
 near-contract-standards = { workspace = true }
 
 sweat-booster-model = { workspace = true }
+regex = "1.10.5"
```

Comment:
Do you need `regex` for the contract of only for tests? Remove this if only for tests.